.gitlab-ci-check-apidocs.yml

```diff
@@ -44,6 +44,7 @@ test:apidocs:verify-swagger:
   image: node:alpine
   script:
     # Verify that the Swagger docs are valid
+    - npm i -g @apidevtools/swagger-cli
```

Comment:
that should be in the before script, right? (nit pick)

Reply:
this is what `npx` does internally, so it was always part of `script` 🤷‍♂️